content/millennium/r4/care-provision/service-request.md

```diff
@@ -0,0 +1,122 @@
+* [Quantity](https://www.hl7.org/fhir/servicerequest-definitions.html#ServiceRequest.quantity_x_){:target="_blank"}
+* [Patient](https://www.hl7.org/fhir/servicerequest-definitions.html#ServiceRequest.subject){:target="_blank"}
+* [Encounter](https://www.hl7.org/fhir/servicerequest-definitions.html#ServiceRequest.encounter){:target="_blank"}
+* [Occurrence](https://www.hl7.org/fhir/servicerequest-definitions.html#ServiceRequest.occurrence_x_){:target="_blank"}
+* [As needed](https://www.hl7.org/fhir/servicerequest-definitions.html#ServiceRequest.asNeeded_x_){:target="_blank"}
+* [Authored on Date](https://www.hl7.org/fhir/servicerequest-definitions.html#ServiceRequest.authoredOn){:target="_blank"}
+* [Requester](https://www.hl7.org/fhir/servicerequest-definitions.html#ServiceRequest.requester){:target="_blank"}
+* [Performer](https://www.hl7.org/fhir/servicerequest-definitions.html#ServiceRequest.performer){:target="_blank"}
+* [Location References](https://www.hl7.org/fhir/servicerequest-definitions.html#ServiceRequest.locationReference){:target="_blank"}
+* [Reason Code](https://www.hl7.org/fhir/servicerequest-definitions.html#ServiceRequest.reasonCode){:target="_blank"}
+* [Body site](https://www.hl7.org/fhir/servicerequest-definitions.html#ServiceRequest.bodySite){:target="_blank"}
+* [Notes](https://www.hl7.org/fhir/servicerequest-definitions.html#ServiceRequest.note){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:service_request, :r4) %>
+
+## Search
+
+Search for ServiceRequests that meet supplied query parameters:
+
+    GET /ServiceRequest?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: true, system: true) %>
+
+### Parameters
+
+ Name      | Required?          | Type          | Description
+-----------|--------------------|---------------|-----------------------------------------------------------------------------------------------------
+ `_id`                    | This, or `patient` or `subject` | [`token`]     | The logical resource id associated with the resource. Example: `12345`
+ `patient`                | This, or `_id` or `subject`     | [`reference`] | The specific patient to return ServiceRequests for. Example: `12345`
+ `subject`                | This, or `_id` or `patient`     | [`reference`] | The subject associated with the ServiceReuest. Must represent a Patient resource. May use the `:Patient` modifier. Example: `subject=Patient/1316020` or `subject:Patient=1316020`
+ `_lastUpdated`           | N                  | [`date`]      | An explicit or implied date-time range within which the most recent clinically relevant update was made to the service request. Must include a time, and must be prefixed by `ge` or `le`. Example: `ge2014-05-19T20:54:02.000Z`
+ `_count`                 | N                  | [`number`]    | The maximum number of ServiceRequests to retrieve in a page. Example: `50`
+
```

Comment:
Can we add a header 'Notes:' here and lets also add the following pointer along with the _lastUpdated related  notes already existing below.
```
* If `_id` is provided, no other parameters may be provided.
```

Reply:
You can find the changes here: https://github.com/cerner/fhir.cerner.com/pull/389/commits/864148c6029d87dff09205f010eaf24ddb7a284e